openwrt-openwisp-monitoring/Makefile

```diff
@@ -0,0 +1,77 @@
+# openwisp.org
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openwrt-openwisp-monitoring
+PKG_VERSION:=$(shell cat ../VERSION)
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)/$(PKG_SOURCE_SUBDIR)
+PKG_LICENSE:=GPL-3.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openwisp-monitoring/default
```

Comment:
The `default` is not needed`and so we also do not need this line `package/openwisp-monitoring=$(call Package/openwisp-monitoring/default)` This is also true for `netjson-monitoring`